src/main/java/epicsquid/roots/EventManager.java

```diff
@@ -54,36 +57,37 @@
   @SubscribeEvent
   public void onBlockHarvested(HarvestDropsEvent event){
     if (event.getHarvester() != null){
-      if (event.getHarvester().getHeldItem(EnumHand.MAIN_HAND) != ItemStack.EMPTY){
-        if (event.getHarvester().getHeldItem(EnumHand.MAIN_HAND).getItem() instanceof ItemKnife){
+      ItemStack tool = event.getHarvester().getHeldItem(EnumHand.MAIN_HAND);
+      if (tool != ItemStack.EMPTY){
```

Comment:
Small thing, but you should use ItemStack.isEmpty() (or in this case, tool.isEmpty()) to check rather than a direct comparison, since it turns out there is actually a small difference (which I found while working on Eio)

Reply:
Just for reference isEmpty() returns empty for otherwise valid stacks that have 0 count in addition to a few other checks (including direct comparison to EMPTY, invalid item damage, the raw item being air, etc.

Reply:
Will do! Thanks for the info